### PR TITLE
chore: add L1 example to lakeformation readme

### PR DESCRIPTION
**Pull request by scanlonp**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk-lib/aws-lakeformation/README.md [hunk 1]
```diff
@@ -25,3 +25,87 @@ For more information on the resources and properties available for this service,
...
+        { 
+          dataLakePrincipalIdentifier: this.formatArn({
+            service: 'iam',
+            resource: 'role',
+            region: '',
+            account: accountId,
+            resourceName: 'Admin',
+          }),
+        },
+        { dataLakePrincipalIdentifier: `arn:aws:iam::${accountId}:role/cdk-hnb659fds-cfn-exec-role-${accountId}-${region}` },
```
Comment (colifran) [inline packages/aws-cdk-lib/aws-lakeformation/README.md:78]: Probably cleaner like this:

```ts
{
  // The CDK cloudformation execution role.
  dataLakePrincipalIdentifier: synthesizer.cloudFormationExecutionRoleArn.replace('${AWS::Partition}', 'aws'),
}
```

File: packages/aws-cdk-lib/aws-lakeformation/README.md [hunk 2]
```diff
@@ -25,3 +25,87 @@ For more information on the resources and properties available for this service,
...
+import { Construct } from 'constructs';
+import { Table, Database, DataFormat, Schema } from '@aws-cdk/aws-glue-alpha';
+import { CfnDataLakeSettings, CfnTag, CfnTagAssociation } from 'aws-cdk-lib/aws-lakeformation';
+
+export class LakeFormationTaggingStack extends cdk.Stack {
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    const accountId = process.env.CDK_DEFAULT_ACCOUNT!;
+    const region = process.env.CDK_DEFAULT_REGION!;
```
Comment (colifran) [inline packages/aws-cdk-lib/aws-lakeformation/README.md:44]: We can probably remove these and use `this.account` and `this.region` everywhere

File: packages/aws-cdk-lib/aws-lakeformation/README.md [hunk 3]
```diff
@@ -25,3 +25,87 @@ For more information on the resources and properties available for this service,
 (Read the [CDK Contributing Guide](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and submit an RFC if you are interested in contributing to this construct library.)
 
 <!--END CFNONLY DISCLAIMER-->
+
+### Example
+
+Using the lakeformation L1s is not always the most straight forward. Here is an example of creating a glue table and putting lakeformation tags on it. Note: this example uses deprecated constructs and overly permissive IAM roles. This example is meant to give a general idea of using the L1s; it is not production level.
```
Comment (colifran) [inline packages/aws-cdk-lib/aws-lakeformation/README.md:31]: Nit: I would remove the first sentence and just start with "Here is an example of creating a glue table and ...". Also should we give an example that isn't using the deprecated glue `Table`?

**Review by colifran — APPROVED** [review]

**Review by vinayak-kukreja — APPROVED** [review]

**Comment by vinayak-kukreja** [comment]
@Mergifyio update

**Pull request merged**